Source/Scene/Cesium3DTileset.js

```diff
@@ -1043,7 +1047,7 @@ function Cesium3DTileset(options) {
         }
         for (let i = 0; i < extraCredits.length; ++i) {
           const credit = extraCredits[i];
-          credits.push(new Credit(credit.html, credit.showOnScreen));
+          credits.push(new Credit(credit.html, this._showCreditsOnScreen));
```

Comment:
This is inside a closure and should be using `that` (declared above)

```suggestion
          credits.push(new Credit(credit.html, that._showCreditsOnScreen));
```